docs/docs/default-actions.mdx

```diff
@@ -187,3 +187,39 @@ Overriding the default action for forms should **only** be used during the proce
 migrating from Rasa Open Source 1 to 2. In this case you can override the default
 action to instruct Rasa Open Source to use the deprecated `FormAction` which is part of
 the Rasa SDK.
+
+## action_unlikely_intent
+
+Rasa Open Source triggers `action_unlikely_intent` via [`IntentTEDPolicy`](./policies.mdx#intenttedpolicy).
+You can control how often this action is predicted by tuning the [`tolerance`](./policies.mdx#intenttedpolicy)
+parameter of `IntentTEDPolicy`.
+
+### Customization
```

Comment:
`action_unlikely_intent` will be predicted however, it will be ignored for featurization by all policies which use only story data. So, effectively it will only be considered by `RulePolicy` in its predictions.

Reply:
@dakshvar22 ,

Do I understand correctly that:
-  `action_unlikely_intent` cannot be used in stories? 
- If you do not have a rule for it either, it is just ignored.

I did not read that anywhere, or I might have just missed that detail.

Reply:
> `action_unlikely_intent` cannot be used in stories?

Yes, it can only appear in rules for training. At inference time, `IntentTEDPolicy` can predict it at anytime but will be ignored by policies which are trained only on stories, for example - `TEDPolicy`, `IntentTEDPolicy`, `MemoizationPolicy`, etc.. This is done to ensure that inference does not break because of a presence of `action_unlikely_intent` at inference time.

> If you do not have a rule for it either, it is just ignored.

Effectively yes. No policy's predictions are expected to be affected by its presence in that case.

Reply:
I've included this information in the description now.